Remotion/Web/Development.WebTesting.ControlObjects/WebButtonControlObject.cs

```diff
@@ -50,6 +53,21 @@ public string GetText ()
       return Scope.Text.Trim();
     }
 
+    /// <summary>
+    /// Returns the button type of the button.
+    /// </summary>
+    public WebButtonType GetButtonType ()
```

Comment:
Please switch to data annotion for this. by using the css class we take a dependency and would actually break if someone where to indeed override the property on the WebButton.

Reply:
The rendering as well? Or add a data annotation in addition to the css class.

Styling is a lot more readable and easier if it is done with a css class so I would prefer not to remove it.

Reply:
In addition. there's the css class for styling and the data-annation that's rendered when the diag-features are turned on.